en/micro-integrator/docs/setup/user_stores/setting_up_ro_ldap.md

```diff
@@ -161,3 +161,15 @@ If you are already using a JDBC user store (database) with another WSO2 product
 	class = "org.wso2.micro.integrator.security.user.core.jdbc.JDBCUserStoreManager"
 	```
 	The datasource configured under the `[[datasource]]` toml heading will now be the effective user store for the Micro Integrator.
+
+## Disabling lazy loading of user store configurations (Optional)
+
+By default, the Micro Integrator initializes the user store configurations on demand (lazy loading). That is, user store configurations are initialized only when the configurations are engaged for the first time and not when the server starts. This behaviour improves the server startup time; however, the first server request that engages user store configurations will experience a delay due to the initialization.
+
```

Comment:
Shall we convey message that it will effect only for one request of the entire life of the server.
```suggestion
By default, the Micro Integrator initializes the user store configurations on demand (lazy loading). That is, user store configurations are initialized only when the configurations are engaged for the first time and not when the server starts. This behaviour improves the server startup time; however, the first server request that engages user store configurations (only effect one request to the entire lifetime of the server) will experience a delay due to the initialization.

```